### ISSUE TITLE
About the circle loss

### ISSUE THREAD

**Issue by HWLMX** [XuyangBai/D3Feat.pytorch#14]
Hi,
Thank you for your excellent work.
I wonder what is the circle loss in  code，the paper only mentions triplet loss and contractive loss.

**Comment by XuyangBai** [comment]
Sorry I mentioned it in the TensorFlow repo:

We have found that [circle loss](https://arxiv.org/abs/2002.10857) provides an insightful idea for metric learning area and shows better and fast convergence for training D3Feat.
